_data/projects/.net-compiler-platform-roslyn.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: .NET Compiler Platform ("Roslyn")
+desc: The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic
+  compilers with rich code analysis APIs.
+site: https://github.com/dotnet/roslyn
+tags:
+- roslyn
+- csx
+- script
+- compiler
+- C#
+- VB.NET
```

Comment:
Nice spot.

This is the output from the migration script (which is using the YAML library that Ruby ships with). So I'd hazard a guess that both formats are valid ways to represent collections in YAML.